evm/rlp/headers.py

```diff
@@ -126,15 +134,15 @@ def hex_hash(self):
 
     @classmethod
     def from_parent(cls,
-                    parent,
-                    gas_limit,
-                    difficulty,
-                    timestamp,
-                    coinbase=ZERO_ADDRESS,
-                    nonce=None,
-                    extra_data=None,
-                    transaction_root=None,
-                    receipt_root=None):
+                    parent: 'BlockHeader',
+                    gas_limit: int,
+                    difficulty: int,
+                    timestamp: int,
+                    coinbase: bytes = ZERO_ADDRESS,
+                    nonce: bytes = None,
+                    extra_data: bytes = None,
```

Comment:
I'm seeing inconsistent spacing around the `=` in these function/method signatures.  Can you update all of these to use the same style.  Preference is no spaces around the `=`

```python
def my_method(self,
              arg1: int=None) -> None:
    ...
```

Reply:
Sorry, for that again. Should be fixed now. I guess there should be some linter for that to have this blow up in CI directly. Will look into it.

Reply:
There's [pylint](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements). Would you be open to use that if it turns out we could configure it to help eliminate such inconsistencies?

Reply:
Feels like something that our current linter `flake8` should have caught or should be able to catch.  Wondering if there's a setting we need to enable/disable.

Reply:
Will look into it

Reply:
Another alternative is https://github.com/ambv/black/, which we could use (locally, when developing) to automatically format our code (with a CI check to ensure everyone is using it), although it's not configurable so we'd need to adopt its chosen style

Reply:
https://github.com/ethereum/py-evm/issues/476